daisy/workflow/workflow.go

```diff
@@ -321,7 +380,7 @@ func (w *Workflow) populate() error {
 	// so Vars replacement must run before this to resolve the final
 	// value for those fields.
 
-	autovars = map[string]string{
+	for k, v := range map[string]string{
 		"NAME":        w.Name,
```

Comment:
Would it be cleaner to do

w.autovars["NAME"] = w.Name
w.autovars["ZONE"] = w.Zone
...